opencti-platform/opencti-front/src/private/components/data/ToolBar.js

```diff
@@ -303,6 +309,44 @@ class ToolBar extends Component {
     this.setState({ displayPromote: false });
   }
 
+  handleCopy() {
+    let selectedValues = [];
+    if (this.props.selectAll) {
+      const observablesTypes = R.map((o) => o.id, this.props.filters.entity_type);
+      const filtering = R.map((o) => o, this.props.filters);
+      if (observablesTypes[0] === 'Stix-Cyber-Observable') {
```

Comment:
It observablesTypes = Stix-Cyber-Observable, it means there are no filters on types (because all the entities are stix-cyber-observables), so we can delete it. If observablesTypes != stix-cyber-observable, it means types filters have been selected, and we keep it [for instance: observablesTypes = ['Bank account', 'Email adress']

Reply:
Ok ! Crystal clear, thanks